CHANGELOG.md

```diff
@@ -45,8 +45,10 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Getting security exception due to access denied 'java.lang.RuntimePermission' 'accessDeclaredMembers' when trying to get snapshot with S3 IRSA ([#4469](https://github.com/opensearch-project/OpenSearch/pull/4469))
 - Fixed flaky test `ResourceAwareTasksTests.testTaskIdPersistsInThreadContext` ([#4484](https://github.com/opensearch-project/OpenSearch/pull/4484))
 - Fixed the ignore_malformed setting to also ignore objects ([#4494](https://github.com/opensearch-project/OpenSearch/pull/4494))
-- [Bug]: Fixed invalid location of JDK dependency for arm64 architecture([#4613](https://github.com/opensearch-project/OpenSearch/pull/4613))
+- Updated jackson to 2.13.4 and snakeyml to 1.32 ([#4556](https://github.com/opensearch-project/OpenSearch/pull/4556))
+- Fixed day of year defaulting for round up parser ([#4627](https://github.com/opensearch-project/OpenSearch/pull/4627))
 - Fixed the SnapshotsInProgress error during index deletion ([#4570](https://github.com/opensearch-project/OpenSearch/pull/4570))
+- [Bug]: Fixed invalid location of JDK dependency for arm64 architecture([#4613](https://github.com/opensearch-project/OpenSearch/pull/4613))
```

Comment:
Thanks @Vishalks for this change. 

Why are we changing the existing changelog entries ?

Reply:
I believe that's a result of resolving the conflict between main and 2.x branches.
I don't think it's a big problem here, he only added one line to this part `Updated jackson to 2.13.4 and snakeyml to 1.32`, which existed in this location in `main` branch, but existed in a below line in this file of `2.x` branch.

Reply:
Of course the changelog in `2.x` branch is very confusing, anyway it will be refactored to be more clear soon.

Reply:
I need to change my way of resolving merge conflicts which is leading to changing the unrelated entries.